proto/docs/p4runtime.md

```diff
@@ -319,3 +319,123 @@ should take into account the top-level message and act as follows:
   switch only returns the counter state associated with the match given in
   `DirectCounterEntry.TableEntry.match`, and does not populate other fields
   within the `TableEntry` (such as `Action`).
+
+## Error reporting
+
+When the p4.Write() fails for some reason, P4 Runtime returns `grpc::Status` to
+report errors. `grpc::Status` has top-level canonical `error_code` that
```

Comment:
has a top-level canonical `error_code` field that...

Reply:
Done.